cli/print/vm_ports.go

```diff
@@ -31,17 +32,17 @@ const (
 )
 
 func VMPorts(outputFormat string, w *tabwriter.Writer, ports []*types.VMPort, header bool) error {
```

Comment:
Get rid of the unusued `w` tabwriter var if we're not going to use it across all funcs in this PR